cumulusci/cumulusci.yml

```
@@ -553,6 +553,7 @@ tasks:
 
 flows:
     ci_beta:
+        group: Continous Integration
```

```suggestion
        group: Continuous Integration
```
(and same for the other copies)